darwin/client.py

```diff
@@ -1,10 +1,10 @@
 import os
 import time
 from pathlib import Path
-from typing import Any, Dict, Iterator, Optional, Union
+from typing import Any, Dict, Iterator, Optional, Tuple, Union, List, Callable
+from darwin.types import Team
```

Comment:
Hate to say this, but you have some issues with your IDE which I think require immediate attention, so I don't bother you all the time with this kind of comments.